src/java.base/share/classes/java/util/JapaneseImperialCalendar.java

```diff
@@ -1364,38 +1345,33 @@ public int getActualMaximum(int field) {
                     CalendarDate d = gcal.newCalendarDate(TimeZone.NO_TIMEZONE);
                     d.setDate(date.getNormalizedYear(), BaseCalendar.JANUARY, 1);
                     if (fd < transition) {
-                        value = (int)(transition - gcal.getFixedDate(d));
+                        value = (int) (transition - gcal.getFixedDate(d));
                     } else {
                         d.addYear(+1);
-                        value = (int)(gcal.getFixedDate(d) - transition);
+                        value = (int) (gcal.getFixedDate(d) - transition);
                     }
                 } else {
-                    LocalGregorianCalendar.Date d = jcal.getCalendarDate(Long.MAX_VALUE,
-                                                                         getZone());
+                    LocalGregorianCalendar.Date d = jcal.getCalendarDate(Long.MAX_VALUE, getZone());
                     if (date.getEra() == d.getEra() && date.getYear() == d.getYear()) {
                         long fd = jcal.getFixedDate(d);
                         long jan1 = getFixedDateJan1(d, fd);
-                        value = (int)(fd - jan1) + 1;
+                        value = (int) (fd - jan1) + 1;
                     } else if (date.getYear() == getMinimum(YEAR)) {
                         CalendarDate d1 = jcal.getCalendarDate(Long.MIN_VALUE, getZone());
                         long fd1 = jcal.getFixedDate(d1);
                         d1.addYear(1);
                         d1.setMonth(BaseCalendar.JANUARY).setDayOfMonth(1);
                         jcal.normalize(d1);
                         long fd2 = jcal.getFixedDate(d1);
-                        value = (int)(fd2 - fd1);
+                        value = (int) (fd2 - fd1);
                     } else {
                         value = jcal.getYearLength(date);
                     }
                 }
             }
```

Comment:
Pull value assignment out of switch?

Reply:
This is a much bigger change which is probably harder to review. I did it, please take a look. One point is whether to unwrap the final `else` after `yield`:
```java
if (...) {
    ...
    yield ...
} else { // should we remove else?
    ...
}
```
I prefer unwrapping, as this reduces the indentation, so I did it. Please tell me if this contradicts with the preferred OpenJDK style.